website/docs/cli/commands/init.html.md

```diff
@@ -36,12 +36,16 @@ The following options apply to all of (or several of) the initialization steps:
 * `-input=true` Ask for input if necessary. If false, will error if
   input was required.
 
-* `-lock=false` Disable locking of state files during state-related operations.
+* `-lock=false` Disable locking of state files during state-related operations
+
+    -> Note: This option was removed in Terraform 0.15.
```

Comment:
Thank you for calling this out! I did two things. First, I added the note into the bullet text. This way, it doesn't interrupt the flow of the page as much and users are much more likely to see/read it when they are reading the actual description of the command. I also added a link to where we explain more about why we removed this/what it did. 

```suggestion
* `-lock=false` ([removed in Terraform 0.15](/upgrade-guides/0-15.html#other-minor-command-line-behavior-changes)) Disable locking of state files during state-related operations.
```